pom.xml

```diff
@@ -16,19 +16,26 @@
 
     <!-- skip javah execution for java9+, see profile below. -->
     <javah.skip>false</javah.skip>
+    <jar.default.skip>false</jar.default.skip>
+    <cmake.generate.skip>false</cmake.generate.skip>
+    <cmake.compile.skip>false</cmake.compile.skip>
+    <!-- TODO: replace with os.detected.bitness for os-maven-plugin 1.6.3 onwards. -->
+    <os.bitness>${sun.arch.data.model}</os.bitness>
```

Comment:
@bmhm Just found https://github.com/trustin/os-maven-plugin/pull/41.  Thanks kindly for submitting an upstream patch for this.  For future simplicity, replacing it now with a similar `TODO:` so it'll be a one-liner when merged.